src/core_plugins/tile_map/public/editors/_tooltip_formatter.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
+
+export function TileMapTooltipFormatterProvider($compile, $rootScope) {
```

Comment:
we will need to rewrite this in react at some point (no need to do this in this PR)